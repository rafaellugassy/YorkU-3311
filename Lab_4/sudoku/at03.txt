set_number(3, 2, 1)

-- Error: invalid column number
set_number(2, 2, 0)
set_number(2, 2, 5)