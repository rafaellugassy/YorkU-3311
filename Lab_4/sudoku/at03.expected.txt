  Setting up a new grid
  0   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(3,2,1)
  Setting up a new grid
  0   0   0   0
  3   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(2,2,0)
  Error: Invalid column number
  0   0   0   0
  3   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(2,2,5)
  Error: Invalid column number
  0   0   0   0
  3   0   0   0
  0   0   0   0
  0   0   0   0
